app/build.gradle

```diff
@@ -13,7 +13,7 @@ android {
 
   defaultConfig {
     applicationId "com.mapbox.services.android.navigation.testapp"
-    minSdkVersion androidVersions.minSdkVersion
+    minSdkVersion 18
```

Comment:
instead of bumping the min sdk for UiAutomator, I would suggest creating an AndroidManifest.xml file in `src>androidTest>AndroidManifest.xml` with the following contents:

```
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:tools="http://schemas.android.com/tools"
          package="com.mapbox.mapboxsdk">
    <uses-sdk tools:overrideLibrary="android.support.test.uiautomator.v18"/>
</manifest>
```

it will crash when you run on api level lower as 18 though other requirements even require higher api levels.